MEMORY_CHECKING.md

```diff
@@ -0,0 +1,21 @@
+# Memory Checking
+
+For some background see the writeup in <https://github.com/pgcentralfoundation/pgrx/issues/1217>.
+
+## Running with Memory Checking
+
+### Valgrind
+
+1. Install valgrind, headers, libs (on Fedora `sudo dnf install valgrind valgrind-devel valgrind-tools-devel` is enough).
+
+2. Run `cargo pgrx init --valgrind`. The only major downside to using this as your primary pgrx installation is that its slow, but you can still run without valgrind.
```

Comment:
I don't understand why these two clauses are simultaneously true ?_?

I suppose it's not critical that I do, but